### ISSUE TITLE
export statements prevent julia-snail-parser-includes from detecting some included files

### ISSUE THREAD

**Issue by orialb** [gcv/julia-snail#20]
Given a module:
```julia
module MyModule

export myfunc

include("file1.jl")
include("file2.jl")
end
```
eval'ing `(julia-snail-parser-includes (get-buffer (current-buffer)))` shows that `file1.jl` is not recognized as an included file, while `file2.jl` is parsed as an included file. 

If I remove the export statement or comment it out it seems that `julia-snail-parser-includes`, recognizes all the included files correctly. In addition the parsing seems to also work if I add a comment or a function definition between the `export` statement and the `include` statement.

**Comment by gcv** [comment]
Parser bug. Should be fixed in 3e57c07, will be in MELPA soon.

**Comment by orialb** [comment]
Thanks for the quick fix. I can confirm it is working for me.
